org.eclipse.winery.repository.ui/src/app/section/section.component.ts

```diff
@@ -153,6 +155,21 @@ export class SectionComponent implements OnInit, OnDestroy {
     onRemoveElement() {
     }
 
+    applyNamespace() {
+        localStorage.setItem('initNamespace', this.initNamespaceString);
+        this.newComponentNamespace = this.applyToscaTypeToNamespace(this.initNamespaceString);
+    }
+
+    onAddModalShown() {
+        this.initNamespaceString = localStorage.getItem('initNamespace') !== null ? localStorage.getItem('initNamespace') : '';
+        this.newComponentNamespace = this.applyToscaTypeToNamespace(this.initNamespaceString);
+    }
+
+    private applyToscaTypeToNamespace(namespace: string) {
+        return namespace.endsWith('/') ? namespace + this.toscaType.toString() :
```

Comment:
what do you return here?

Reply:
and you don't need to do `.toString()` on a `ToscaType` because it is a string enum